joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/ImportTests.scala

```diff
@@ -133,6 +133,77 @@ class ImportTests extends RubyCode2CpgFixture(withPostProcessing = true) with In
     }
   }
 
+  "implicitly imported types in base class that are qualified names" should {
```

Comment:
Looks to me like you only added test cases for the already existing scenario without external types. Please also add tests with external types.